docs/source/building_an_experiment.rst

```diff
@@ -2,49 +2,306 @@
+necessary slice keys.
+
+Slice Keys
+----------
+
+These are the keys that are set by the user when initializing a slice. Some 
+are required, some can be defaulted, and some are set by the experiment 
+and are read-only.
+
+**Slice Keys Required by the User**
+
+pulse_sequence *required*
+    The pulse sequence timing, given in quantities of tau_spacing, for example
+    normalscan = [0, 14, 22, 24, 27, 31, 42, 43]. 
+
+tau_spacing *required*
+    multi-pulse increment in us, Defines minimum space between pulses.
+
+pulse_len *required*
+    length of pulse in us. Range gate size is also determined by this.
+
+num_ranges *required*
+    Number of range gates.
+
+first_range *required*
+    first range gate, in km
+
+intt *required or intn required*
+    duration of an integration, in ms. (maximum)
+
+intn *required or intt required*
+    number of averages to make a single integration, if intt = None.
+
+beam_angle *required*
+    list of beam directions, in degrees off azimuth. Positive is E of N. Array
+    length = number of beams. Traditionally beams have been 3.24 degrees separated but we
+    don't refer to them as beam -19.64 degrees, we refer as beam 1, beam 2. Beam 0 will
+    be the 0th element in the list, beam 1 will be the 1st, etc. These beam numbers are
+    needed to write the beam_order list. This is like a mapping of beam number (list
+    index) to beam direction off boresight.
+
+beam_order *required*
+    beam numbers written in order of preference, one element in this list corresponds to
+    one integration period. Can have lists within the list, resulting in multiple beams
+    running simultaneously in the averaging period, so imaging. A beam number of 0 in
+    this list gives us the direction of the 0th element in the beam_angle list. It is
+    up to the writer to ensure their beam pattern makes sense. Typically beam_order is
+    just in order (scanning W to E or E to W, ie. [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
+    11, 12, 13, 14, 15]. You can list numbers multiple times in the beam_order list,
+    for example [0, 1, 1, 2, 1] or use multiple beam numbers in a single
+    integration time (example [[0, 1], [3, 4]], which would trigger an imaging
+    integration. When we do imaging we will still have to quantize the directions we
+    are looking in to certain beam directions.
+
+clrfrqrange *required or txfreq or rxfreq required*
+    range for clear frequency search, should be a list of length = 2, [min_freq, max_freq]
+    in kHz. **Not currently supported.**
+
+txfreq *required or clrfrqrange or rxfreq required*
+    transmit frequency, in kHz. Note if you specify clrfrqrange it won't be used.
+
+rxfreq *required or clrfrqrange or txfreq required*
+    receive frequency, in kHz. Note if you specify clrfrqrange or txfreq it won't be used. Only
+    necessary to specify if you want a receive-only slice.
+
+
+**Defaultable Slice Keys**
+
+acf *defaults*
```

Comment:
Does a default of false here make sense? I can point to the epopsound program as experience that it doesn't make sense to default to acfs being off. We  ran for a while before Gareth actually asked, at which point I had to disappoint him, because I had defaulted acfs to not be generated.

Reply:
I think we could change this eventually if you like, but it will be a more complex default because it can only be True on certain conditions and I think we have yet to set up those conditions (pulse encoding not worked into acfs for example, and I think there is other things to consider). So it would be more confusing to understand, ie. "True on default if x, y , and z otherwise False"